### PR TITLE
Usernames Misc: Bulk Email and New Party Modal

### PR DESCRIPTION
**Pull request by SabreCat**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # here, if applicable. This will automatically close the issue if your PR is merged in)

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

* Updates the bulk email script to use Mongoose models and sets it up as a reusable template.
* Edits the "New Party" dialog to use Usernames instead of UUIDs, including some style updates from Zeplin.
* Archives several old-style migrations that still used `monk`.

### CODE REVIEW

File: website/client/components/groups/createPartyModal.vue [hunk 1]
```diff
@@ -61,140 +75,120 @@ b-modal#create-party-modal(title="Empty", size='lg', hide-footer=true)
...
+    background-color: $white;
+    border: 0px;
+    border-radius: 0px;
+    color: $gray-300;
+    padding: 0rem 0.1rem 0rem 0.75rem;
   }
 
   .join-party {
-    background-image: url('~client/assets/images/party@3x.png');
+    background-image: url('~client/assets/images/party.png');
```
Comment (paglias) [inline website/client/components/groups/createPartyModal.vue:115]: is this going to look bad on hd displays?

**Review by paglias — COMMENTED** [review]
just one comment on an image

File: website/client/components/groups/createPartyModal.vue (hunk 1 above)
Comment (SabreCat) [reply website/client/components/groups/createPartyModal.vue:115]: At the moment it looks bad at 1920x1080, so I'm not sure which way is preferable..

Comment (paglias) [reply website/client/components/groups/createPartyModal.vue:115]: Have you tried the @3x version?

Comment (SabreCat) [reply website/client/components/groups/createPartyModal.vue:115]: Yep, that's what it was before. It scales down and gets blurry.

Comment (paglias) [reply website/client/components/groups/createPartyModal.vue:115]: Hmm it looks like that with higher def images you have to specify the background-size https://stackoverflow.com/questions/16154494/retina-displays-high-res-background-images to the original size, but if it looks good, i'd say it's fine

**Pull request merged**